### PR TITLE
landscape broken change media queries

### LINKED ISSUE #260: Responsive design broken in landscape mode
## iPhone SE portrait mode (works)
2018-05-29-124719_776x1256_scrot

## iPhone SE landscape mode (broken)
2018-05-29-124653_1339x878_scrot

### PR DESCRIPTION
**Pull request by jalbertsr**
Close #260 

- removed min-width 
- max-width set to 1180px, iphoneX in landscape mode is 812 (we support form iphone5 to iphoneX)
<br/><br/><br/><url>LiveURL: https://smartfaq-270-landscape-broken.surge.sh</url>

### CODE REVIEW

File: src/IntroPage/index.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const style = css`
     margin-left: 182px;
     line-height: 1.4;
   }
-  @media only screen and (min-width: 320px) and (max-width: 480px) {
+  @media only screen and (max-width: 480px) {
```
Comment (jalbertsr) [inline src/IntroPage/index.js]: since we don't have designs for landscape mode yet, what's better to show in landscape mode, mobile or desktop?

Comment (kantord) [reply src/IntroPage/index.js]: Desktop if landscape screen has enough width. Mobile otherwise.

**Comment by kantord** [comment]
2018-06-06-133955_2019x1364_scrot

Still broken on Nexus 7

**Comment by jalbertsr** [comment]
https://github.com/kiwicom/smart-faq/pull/345#issuecomment-395039284 kill me, btw.. are we covering nexus 7 landsacpe for now?? 960px... 🙄 looks more like a laptop than a mobile...

**Comment by kantord** [comment]
2018-06-06-134205_1810x894_scrot

This seems to be usable (width 813), however, it breaks at width 814 :thinking: 

Seems to be usable again around width 1198:

2018-06-06-134405_2557x977_scrot

BTW this is from the responsive debug tool in Firefox

**Comment by kantord** [comment]
I think trying to cover specific models is futile, although we might pay special attention to popular devices. What I would focus on is more like having a usable layout (no overlaps, no unreadable text, no unclickable buttons etc.) on each screen width between, say, 240 to our `<html>`'s `max-width`. (Or if we don't have max width then the biggest number frontend officially supports. For example a width for a 4K monitor)

**Comment by jaroslav-kubicek** [comment]
LGTM, just rebase and merge :)

**Pull request merged**